### DISCUSSION TITLE
[IDEA] Make "Listeners" not part of regular allowed number of users and...

### DISCUSSION THREAD

**Discussion by 38github** [void-linux/void-packages#50403] [General]
... make them use less resources like less bitrate, larger framesize, etc and lower priority to minimize CPU and network usage. Would like be something that is of interest to implement?

Maybe also make regular users that are muted for a certain amount of time not able to be a regular user and becomes a "listener user" allowing opening user slot on the server for active participants.

**Comment by classabbyamp** [comment]
did you mean to make this discussion in another repo?

**Reply by 38github** [reply]
This post was not even intended to be posted in this repo. I apologise for this and please remove if you can, thanks.
